kde-apps/audiocd-kio/audiocd-kio-25.08.49.9999.ebuild

```diff
@@ -37,7 +37,7 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
-PATCHES=( "${FILESDIR}/${PN}-19.04.0-handbook.patch" )
+PATCHES=( "${FILESDIR}/${PN}-23.03.80-handbook.patch" )
```

Comment:
It should be package version introduced patch with; you're not going to retroactively fix old and cleaned up versions with that? ;)